macros/models/_sector/transfers/erc20_stablecoins/transfers_erc20_stablecoins.sql

```diff
@@ -0,0 +1,150 @@
+{% macro transfers_erc20_stablecoins(blockchain) %}
+WITH
+    stables_in_tx AS (
+        SELECT 
+            evt_tx_hash
+            , COUNT(*) AS stables_in_tx 
+        FROM (
+            SELECT distinct 
+                t.contract_address
+                , t.evt_tx_hash 
+            FROM {{ source('erc20_' + blockchain, 'evt_transfer') }} t
```

Comment:
rather than read from source transfers again, can we take advantage of the new transfers spells in the `_sector/tokens/` directory?

Reply:
Is it better for me to use the transfers_enrich macro for ethereum or just use the `tokens_ethereum.transfers` as the source?

Reply:
Tired using them and now get: "Error: Query exceeds cluster capacity (memory/pages), please optimize your query"

Reply:
it's better to read from them directly, either as ref or source, not reuse macro. they are heavy tables, especially on larger chains, so not too surprising to see cluster crash. it would be advised to start with one chain in the PR to keep it simple and give cluster more dedicated space to run